enterprise/server-enterprise/src/main/java/org/neo4j/server/enterprise/EnterpriseNeoServer.java

```diff
@@ -82,7 +78,8 @@ public EnterpriseNeoServer( ConfigurationBuilder configurator, Database.Factory
     protected PreFlightTasks createPreflightTasks()
     {
         return new PreFlightTasks( logProvider,
-                new EnsurePreparedForHttpLogging( configurator.configuration() ), new PerformUpgradeIfNecessary(
+                new EnsurePreparedForHttpLogging( configurator.configuration() ),
+                new PerformUpgradeIfNecessary(
                         getConfig(), configurator.getDatabaseTuningProperties(), logProvider,
                         StoreUpgrader.NO_MONITOR ), new PerformRecoveryIfNecessary( getConfig(),
                         configurator.getDatabaseTuningProperties(), logProvider ) );
```

Comment:
Did you miss removing the upgrade preflight task from the EnterpriseNeoServer? You removed it from the CommunityNeoServer.